src/analysis/liouville/inequalities_and_series.lean

```diff
@@ -0,0 +1,122 @@
+/-
+Copyright (c) 2020 Jujian Zhang. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Damiano Testa, Jujian Zhang
+-/
+import algebra.char_p.invertible
+import analysis.specific_limits
+/-!
+
+# Lemmas on inequalities and series for Liouville constants
+
+This file contains lemmas about inequalities and series that are used in the proof to show that
+transcendental Liouville numbers exist.
+-/
+
+open_locale nat big_operators
+open set real
+
+variable {m : ℝ}
+
+/--  An inequality involving `2 : ℝ`. -/
+lemma sub_one_div_inv_le_two (hm : 2 ≤ m) :
```

Comment:
I feel like the first five lemmas could be moved to other files none the less. (And then also generalized when it makes sense, to arbitrary ordered fields or something like that?)

Reply:
Ok, I rolled up my sleeves and scattered them through mathlib!